platform/net/ubuntu_net_manager_test.go

```diff
@@ -904,6 +938,108 @@ iface ethstatic inet static
 				Expect(networkConfig.StringContents()).To(Equal(expectedNetworkConfiguration))
 			})
 		})
+
+		Context("when manual networks were configured with portal IP", func() {
```

Comment:
I don't understand the title for this context? What portal IP are we tabling about? I am guessing you mean `portable` IPs?